ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/ReadService.scala

```diff
@@ -56,11 +56,12 @@ trait ReadService extends ReportsHealth {
     *   and [[Update.PublicPackageUpload]] updates for all packages referenced by
     *   the [[Update.TransactionAccepted]].
     *
-    * - *monotonic record time*: for any update `u1` with an associated record
-    *   time `rt1` before an update `u2` with an associated record time `rt2`
-    *   in the stream, it holds that `rt1 <= rt2`. The updates with an
-    *   associated record time are [[Update.Heartbeat]] and [[Update.TransactionAccepted]],
-    *   which both store the record time in the `recordTime` field.
+    * - *causal monotonicity*: given a [[Update.TransactionAccepted]] with an associated
+    *   record time `rt_tx`, it holds that `rt_tx >= rt_c` for all `c`, where `c` is a
+    *   contract referenced by the transaction and `rt_c` the record time of the
```

Comment:
```suggestion
    *   contract used by the transaction and `rt_c` the record time of the
```
I prefer the term `used` over `referenced` because used is defined in the ledger model. Referencing could be misunderstood to mean that the sole mentioning of a contract ID in a contract argument in the transaction enforces the monotonicity requirement, but we don't want to guarantee that.

Reply:
Done.